userspace/Build.mk

```diff
@@ -130,9 +130,16 @@ build/userspace/$(APP)/$(BOARD)/full_image: \
 	arm-none-eabi-objcopy --update-section \
 		.apps=build/userspace/$(APP)/cortex-m3/cortex-m3.tbf \
 		build/userspace/$(APP)/$(BOARD)/unsigned_image
-	$(TANGO_CODESIGNER) --b --input build/userspace/$(APP)/$(BOARD)/unsigned_image \
-		--key=$(TANGO_CODESIGNER_KEY) \
-		--output=build/userspace/$(APP)/$(BOARD)/signed_image
+	if [ -n "${TANGO_CODESIGNER}" -a -n "${TANGO_CODESIGNER_KEY}" ]; then \
+		$(TANGO_CODESIGNER) --b --input build/userspace/$(APP)/$(BOARD)/unsigned_image \
+			--key=$(TANGO_CODESIGNER_KEY) \
+			--output=build/userspace/$(APP)/$(BOARD)/signed_image; \
+	else \
+		echo "***** Substituting unsigned_image for signed_image -- one of"; \
+		echo "*****   CODESIGNER{,_KEY} is empty-- hope that's OK."; \
```

Comment:
Add space after `empty` and before `--`.

Reply:
If I'm not creating `full_image` when one of CODESIGNER{,_KEY} is not set, there's no point in creating `signed_image` either, so I have changed the comment.

Reply:
Done.